changes-test-results.diff

```diff
@@ -0,0 +1,63 @@
+diff --git a/tests/static_temperature/screen-output b/tests/static_temperature/screen-output
```

Comment:
You accidentally added this file as well. Please remove (`git rm changes-test-results.diff`).